sound/soc/intel/boards/sof_sdw.c

```diff
@@ -325,6 +325,14 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 					RT711_JD2 |
 					SOF_SDW_FOUR_SPK),
 	},
+	/* MeteorLake devices */
+	{
+		.callback = sof_sdw_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_PRODUCT_FAMILY, "Intel_mtlrvp"),
+		},
+		.driver_data = (void *)(RT711_JD1 | SOF_SDW_PCH_DMIC),
```

Comment:
Can you check if the PCH_DMIC is required @yongzhi1? This is an old way of adding dmic information, we now use the dmic_num that's based on the presence of NHLT. Thanks!

Reply:
Ack